client/js/compute.js

```diff
@@ -0,0 +1,578 @@
+const compute_ops = [
+    {
+      name: '=',
+      desc: 'sets the variable to the value of x',
+      sample: 'var a = ${x}',
+      call: function(v, x, y) { return v = y } // y for compatibility with SET
```

Comment:
I now simply `return y`, as that's what it effectively does.

Reply:
The same applies to all other functions.

Reply:
I find the functions easier to read/understand with `return v = `, though. Otherwise you have some functions that return v (e.g. push) and others that don't.

Reply:
If you have a reason for keeping it, sure. I was just pointing out that it doesn't do anything.

For me it's not easier to understand because I'm wondering what it's supposed to accomplish. Just FYI.